stats/src/main/java/io/airlift/stats/QuantileDigest.java

```diff
@@ -125,6 +140,73 @@ public QuantileDigest(QuantileDigest quantileDigest)
         merge(quantileDigest);
     }
 
+    public QuantileDigest(Slice serialized)
+    {
+        SliceInput input = new BasicSliceInput(serialized);
+
+        maxError = input.readDouble();
+        alpha = input.readDouble();
+
+        if (alpha == 0.0) {
+            ticker = noOpTicker();
+        }
+        else {
+            ticker = Ticker.systemTicker();
+        }
+
+        min = input.readLong();
+        max = input.readLong();
+        int nodeCount = input.readInt();
```

Comment:
Can we validate the node count? Given we know the domain, isn't there a max node count?